docs/vagrant.md

```diff
@@ -1,6 +1,6 @@
 # Vagrant
 
-Assuming you have Vagrant 2.0+ installed with virtualbox, libvirt/qemu or
+Assuming you have Vagrant 2.0+ installed with virtualbox (it may work with libvirt/qemu or
```

Comment:
Is that actually correct though ? I think the CI works with vagrant-libvirt, do we test virtualbox anywhere ?
```shell
$ grep -R vagrant-libvirt
test-infra/vagrant-docker/Dockerfile: vagrant plugin install vagrant-libvirt
pipeline.Dockerfile:    && vagrant plugin install vagrant-libvirt \
```

Reply:
You are right, I guess the ` but is untested` only concerned the `vmware` part..

libvirt/qemu is used in ci and vbox is used on local when testing vagrant (for most contributors at least)